media/CircleGraph/view.js

```diff
@@ -528,13 +528,12 @@ view.View = class {
       const container = this._getElementById("graph");
       let x = 0;
       let y = 0;
-      for (const element of selection) {
+      let element = selection[selection.length - 1];
```

Comment:
what is this change for?

Reply:
To make view scroll to the last selected window. To prevent view from sliding to the middle of selection.
See https://github.com/Samsung/ONE-vscode/pull/1505#issuecomment-1439186216

Reply:
I don't think this is a valid change. please do not change this.

Reply:
Please do not change this.

Reply:
ok

Reply:
done. so the issue https://github.com/Samsung/ONE-vscode/pull/1505#issuecomment-1439186216 is back.